packages/server/graphql/public/types/User.ts

```diff
@@ -57,6 +63,50 @@ const User: UserResolvers = {
     const stripeLineItems = await fetchAllLines(invoiceId)
     const invoice = await manager.retrieveInvoice(invoiceId)
     return generateInvoice(invoice, stripeLineItems, orgId, invoiceId, dataLoader)
+  },
+  availableTemplates: async ({id: userId}, {first, after}, {authToken, dataLoader}) => {
+    const viewerId = getUserId(authToken)
+    const user = (await dataLoader.get('users').load(userId))!
+    const teamIds =
+      viewerId === userId || isSuperUser(authToken)
+        ? user.tms
+        : user.tms.filter((teamId: string) => authToken.tms.includes(teamId))
+
+    // Get the team templates.
+    const teamTemplates = await dataLoader
+      .get('meetingTemplatesByType')
+      .loadMany([
+        ...teamIds.map((teamId) => ({teamId, meetingType: 'poker' as MeetingTypeEnum})),
+        ...teamIds.map((teamId) => ({teamId, meetingType: 'retrospective' as MeetingTypeEnum}))
+      ])
+    const scoredTeamTemplates = await getScoredTemplates(teamTemplates.filter(isValid).flat(), 0.9)
```

Comment:
-1 what does `0.9` mean here and why this particular value? it'd be great to either extract a const with some comment so it's easier to understand what's happening here